charts/daemonset/templates/daemonset.yaml

```diff
@@ -11,6 +11,9 @@ spec:
       labels:
         app: {{ include "app.fullname" . }}
       annotations:
+        {{- with .Values.annotations }}
```

Comment:
Do we only want the annotations added to the daemonset?  If thats the case should the values.yaml call that out specifically that these are deployment specific annotations?

Reply:
I believe so. Down the road, adding a common annotation that is applied to all resources is probably a good idea.
In this case, I do have "# -- Annotations to be applied to the daemonset" as a comment in values.yaml, think that is enough context?

Reply:
It likely is enough but if you ever did go for common annotations then would you be making a breaking change to the chart?  If you called this daemonsetAnnotations from the start then you can add common ones when ready without an api change.